Makefile

```diff
@@ -133,7 +133,8 @@ OPERATOR_SDK_VERSION = $(shell $(OPERATOR_SDK) version 2>/dev/null | sed 's/^ope
 OPERATOR_SDK_VERSION_REQ = v1.22.0-ocp
 operator-sdk: ## Download operator-sdk locally if necessary.
 ifneq ($(OPERATOR_SDK_VERSION_REQ),$(OPERATOR_SDK_VERSION))
-	curl https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/4.11.0/operator-sdk-linux-x86_64.tar.gz? | tar -xz -C bin/
+	curl https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/4.13.1/operator-sdk-darwin-x86_64.tar.gz? | tar -xz -C bin/
```

Comment:
This links to operator-sdk v1.28.0-ocp. Line 133 should be update to v1.28.0-ocp too.